Friendly-plans/app/src/main/res/layout/fragment_task_container.xml

```diff
@@ -1,89 +1,99 @@
 <?xml version="1.0" encoding="utf-8"?>
-<GridLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<ScrollView
+  xmlns:android="http://schemas.android.com/apk/res/android"
   xmlns:tools="http://schemas.android.com/tools"
   android:layout_width="match_parent"
-  android:layout_height="match_parent"
-  android:padding="5dp"
-  android:columnCount="5"
-  android:orientation="vertical">
+  android:layout_height="match_parent">
 
-  <TextView
-    android:id="@+id/id_task_create_description"
-    android:layout_columnSpan="5"
-    android:background="#cccccc"
-    android:text="@string/task_create_description"
-    android:textAppearance="?android:attr/textAppearanceLarge"/>
+  <GridLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:padding="5dp"
+    android:columnCount="5"
+    android:orientation="vertical">
 
-  <TextView
-    android:layout_column="0"
-    android:layout_row="1"
-    android:text="@string/task_name"/>
+    <TextView
+      android:id="@+id/id_task_create_description"
+      android:layout_columnSpan="5"
+      android:background="#cccccc"
+      android:text="@string/task_create_description"
+      android:textAppearance="?android:attr/textAppearanceLarge"/>
 
-  <EditText
-    android:id="@+id/id_et_task_name"
-    android:layout_column="1"
-    android:layout_row="1"
-    android:cursorVisible="true"
-    android:inputType="text"
-    android:minWidth="150dp"
-    tools:text="This is task name."/>
+    <TextView
+      android:id="@+id/id_tv_task_name_label"
+      android:layout_column="0"
+      android:layout_row="1"
+      android:text="@string/task_name"/>
 
-  <TextView
-    android:layout_column="0"
-    android:layout_row="2"
-    android:text="@string/task_picture"/>
+    <EditText
+      android:id="@+id/id_et_task_name"
+      android:layout_column="1"
+      android:layout_row="1"
+      android:cursorVisible="true"
+      android:inputType="text"
+      android:minWidth="150dp"
+      tools:text="This is task name."/>
```

Comment:
tools:text is not available for user. Only to see placeholder on preview. No sense to bring to resources.